Index: vendor/paypal/module-braintree-core/Model/Adminhtml/System/Config/CountryCreditCard.php
<+>UTF-8
===================================================================
--- a/vendor/paypal/module-braintree-core/Model/Adminhtml/System/Config/CountryCreditCard.php
+++ b/vendor/paypal/module-braintree-core/Model/Adminhtml/System/Config/CountryCreditCard.php
@@ -69,15 +69,17 @@
     {
         $value = $this->getValue();
         $result = [];
-        foreach ($value as $data) {
-            if (empty($data['country_id']) || empty($data['cc_types'])) {
-                continue;
-            }
-            $country = $data['country_id'];
-            if (array_key_exists($country, $result)) {
-                $result[$country] = $this->appendUniqueCountries($result[$country], $data['cc_types']);
-            } else {
-                $result[$country] = $data['cc_types'];
+        if (is_array($value)) {
+            foreach ($value as $data) {
+                if (empty($data['country_id']) || empty($data['cc_types'])) {
+                    continue;
+                }
+                $country = $data['country_id'];
+                if (array_key_exists($country, $result)) {
+                    $result[$country] = $this->appendUniqueCountries($result[$country], $data['cc_types']);
+                } else {
+                    $result[$country] = $data['cc_types'];
+                }
             }
         }
         $this->setValue($this->serializer->serialize($result));
