Index: vendor/magento/module-cron/Model/Config/Backend/Product/Alert.php
<+>UTF-8
===================================================================
--- a/vendor/magento/module-cron/Model/Config/Backend/Product/Alert.php
+++ b/vendor/magento/module-cron/Model/Config/Backend/Product/Alert.php
@@ -71,8 +71,8 @@
      */
     public function afterSave()
     {
-        $time = $this->getData('groups/productalert_cron/fields/time/value');
-        $frequency = $this->getData('groups/productalert_cron/fields/frequency/value');
+        $time = $this->getData('groups/productalert_cron/fields/time/value') ?: [0,0,0];
+        $frequency = $this->getData('groups/productalert_cron/fields/frequency/value') ?: 'D';

         $cronExprArray = [
             (int)$time[1], //Minute
